### PR TITLE
bug/Sidebar fix

### PR DESCRIPTION
**Pull request by SepidehShahbazi**
Issue #76 is completed:

Sidebar Component:

- Exporting contact icon from Figma.
- Adding it as SVG file.
- Converting the SVG files to React Components called ContactsIcon.tsx, DocumentsIcon.tsx and SettingsIcon.tsx.
- Updating the Sidebar.tsx.

Sidebar

### CODE REVIEW

File: Client/public/assets/icons/sidebar/ContactsIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const ContactsIcon = () => {
```
Comment (mahid797) [inline Client/public/assets/icons/sidebar/ContactsIcon.tsx:1]: Please make all of these component Icons reusable

File: Client/public/assets/icons/sidebar/DocumentsIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const DocumentsIcon = () => {
```
Comment (mahid797) [inline Client/public/assets/icons/sidebar/DocumentsIcon.tsx:1]: Same as above

File: Client/public/assets/icons/sidebar/SettingsIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+const SettingsIcon = () => {
```
Comment (mahid797) [inline Client/public/assets/icons/sidebar/SettingsIcon.tsx:1]: Same as above

**Review by mahid797 — CHANGES_REQUESTED** [review]
Please make all of these component icons reusable so props can be passed to them. You can check my implementation of React Icons in `BluewaveLogo.tsx` or the icons in `public/assets/icons/auth/` as examples.

**Review by mahid797 — APPROVED** [review]

**Review by Paxxack — APPROVED** [review]
Good job!

**Pull request merged**